validate.py

```diff
@@ -116,7 +116,8 @@ def run_checks_on_output(checks_dict: dict[str, CheckData], output_df) -> Checks
     for check_name in checks_dict.keys():
         check = checks_dict[check_name]
         check_func = check['condition']
-        output_df[check_name] = output_df.iloc[:, 1]
+        # If two columns not given, default to first column as the value to check
+        output_df[check_name] = output_df.iloc[:, 1] if output_df.shape[1] > 1 else output_df.iloc[:, 0] 
```

Comment:
I noticed we weren't handling the validation and warning printout for a query of one field, so I added the case for this indicator.

Reply:
True. There were validation queries looking for duplicates in UDW assignments and courses, and those queries turn two column datasets: https://github.com/tl-its-umich-edu/unizin-validation/commit/0825e3a9546f7a6cd1c78a2e0ccc0128c2f93c18